lib/improver/orographic_enhancement.py

```diff
@@ -509,43 +505,24 @@ def _create_output_cubes(self, orogenh_data, reference_cube):
             except KeyError:
                 continue
 
-        orogenh = iris.cube.Cube(
+        orog_cube = iris.cube.Cube(
             orogenh_data, long_name="orographic_enhancement",
             units="mm h-1", attributes=attributes,
             dim_coords_and_dims=[(y_coord, 0), (x_coord, 1)],
             aux_coords_and_dims=aux_coords)
-        orogenh.convert_units("m s-1")
+        orog_cube.convert_units("m s-1")
 
-        # regrid the orographic enhancement cube onto the standard grid and
-        # mask extrapolated points
-        orogenh_standard_grid = orogenh.regrid(
-            reference_cube, iris.analysis.Linear(extrapolation_mode='mask'))
+        for key, value in self.topography.attributes.items():
+            if 'mosg__grid' in key:
+                orog_cube.attributes[key] = value
 
-        for axis in ['x', 'y']:
-            orogenh_standard_grid = sort_coord_in_cube(
-                orogenh_standard_grid, orogenh_standard_grid.coord(axis=axis))
-            orogenh_standard_grid.coord(axis=axis).points = (
-                orogenh_standard_grid.coord(axis=axis).points.astype(
-                    np.float32))
-            if orogenh_standard_grid.coord(axis=axis).bounds is not None:
-                orogenh_standard_grid.coord(axis=axis).bounds = (
-                    orogenh_standard_grid.coord(axis=axis).bounds.astype(
-                        np.float32))
-
-        # add any relevant grid definition attributes
-        for data_cube, grid_cube in zip([orogenh, orogenh_standard_grid],
-                                        [self.topography, reference_cube]):
-            for key, val in grid_cube.attributes.items():
-                if 'mosg__grid' in key:
-                    data_cube.attributes[key] = val
-
-        return orogenh, orogenh_standard_grid
+        return orog_cube
 
     def process(self, temperature, humidity, pressure, uwind, vwind,
                 topography):
         """
-        Calculate precipitation enhancement over orography on standard and
-        high resolution grids.  Input variables are expected to be on the same
+        Calculate precipitation enhancement over orography on high
+        resolution grids.  Input variables are expected to be on the same
         grid (either standard or high resolution).
```

Comment:
This docstring is not helpful.  I'll think about how to make it better.

Reply:
How about:
```
Calculate precipitation enhancement over orography on high resolution
grid.  Input diagnostics are expected to be on the same grid, and are
regridded to match the orography.
```

Reply:
Done.